hack/generate/gen-test-framework.sh

```diff
@@ -13,7 +13,13 @@ cd test/test-framework
 
 # Ensure test-framework is up-to-date with current Go project dependencies.
 echo "$(../../build/operator-sdk print-deps)" > go.mod
-sed -i 's|github.com/operator-framework/operator-sdk\s\+master||g' go.mod
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  sed -i " " 's|github.com/operator-framework/operator-sdk\s\+master||g' go.mod
+else
+  sed -i 's|github.com/operator-framework/operator-sdk\s\+master||g' go.mod
+fi
+
```

Comment:
This should work on both Mac and Linux:
```
sed -i".bak" -E -e "s|github.com/operator-framework/operator-sdk[[:blank:]]+master||g" go.mod; rm -f go.mod.bak
```

Important things to note:
* There is no space between `-i` and the quoted extension
* Use -E to specify POSIX extended regular expressions.
* Use `-e` to specify the sed script.

Ditto for all other usages.